### ISSUE TITLE
Error when playing video files stored on Dune HDD

### ISSUE THREAD

**Issue by wgoldman** [newloran2/emplexer#42]
Thanks for your help and all your efforts on emplexer!  This is a follow-up to an issue I described on the Plex Forum.  My emplexer log files are at https://www.dropbox.com/sh/iel34znlqvqk6y8/_X6oNQPkFH

Here's the description of the problem (copied from the forum posts):  

Is anyone besides me trying to use emplexer to play video files from an internal disk drive within the Dune?  I have the Plex Media Server running on a MacBook Pro, and my movies are on a drive installed in a Dune HD Base 3D.  The emplexer client is able to gather and display the movie metadata from Plex, but none of those movies will actually play on the Dune -- I get a "format not supported" error.  The same movies play properly if I use the normal Dune interface, and they will also play via emplexer If I move the video files to the MacBook Pro.  But emplexer fails to play these files if they are on the Dune's internal drive.

Advice from newloran2 was to switch from nfs to smb for communicating with the Dune.  Unfortunately, the problem remains.  I have tried reinstalling emplexer, restarting Plex, and rebooting all the hardware.  I'm using the latest beta firmware for the Dune.

**Comment by newloran2** [comment]
wgoldman,

I what's wrong in your log.

You not setup your smb on emplexer correctly, see smb://user:password@10.0.1.58/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/.
if you use your dune smb without password you need to to something like that
smb://guest@10.0.1.58/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/.

This is valid if you use one machine to store your files and another to serve PMS (like your situation), but for cases that files are in same dune where emplexer running you able to use a local access.

To do that you need  in smb on SMB Advanced configuration configure you plex folder like that:
/tmp/mnt/storage/Name os your HD.
In your case something like this **/tmp/mnt/storage/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965**

plz if this solve your problem send a feedback on this issue and i will close.

**Comment by wgoldman** [comment]
That works perfectly, newloran2!!  Thank you for your speedy reply!

**Comment by wgoldman** [comment]
A follow-up complication to this issue:

For all .mkv files, the solution above works perfectly.  That is, I can use emplexer to play movie files on my Dune's internal HDD as long as the emplexer path is specified like newloran2 suggested (/tmp/mnt/storage/DuneHDD_. . .).  I can use either smb or nfs to access my Dune HDD .mkv files this way.  

Unfortunately, the same is not true for files that are full-disc rips, like .iso (generated from Blu-Ray) or .dvdmedia (generated from DVD).  If I've selected one of these and press "Enter" on the remote, nothing happens.  If I press "Info" on the remote, the metadata appears on screen along with a "Play" button.  But then selecting "Play" gets me only a "format not supported" error.  The same .iso or .dvdmedia files play properly if I use the normal Dune interface instead of emplexer.

Any thoughts on this?  Thanks in advance.

**Comment by newloran2** [comment]
try this way:
storage_name://storage_name
ex: storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965

**Comment by wgoldman** [comment]
Thanks for the fast response -- I tried setting that path using both smb and nfs.  Unfortunately, the problem persists with my .iso and .dvdmedia files.

**Comment by newloran2** [comment]
Can you plz simulate this bug and send-me your emplexer.log?

**Comment by wgoldman** [comment]
My latest emplexer log files are at:
https://www.dropbox.com/sh/iel34znlqvqk6y8/_X6oNQPkFH

**Comment by newloran2** [comment]
wgoldman,

Sorry for this too delayed answer.

In your log i saw that you have configured some areas as suggested, but you dont use any of this areas.
/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965] => /tmp/mnt/storage/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965
    [screen.emplexer_movie_list.view_idx] => 0
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/3D Movies] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/3D Movies
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/DVD Movies] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/DVD Movies
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/DVD TV Shows] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/DVD TV Shows
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/HD Movies] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/HD Movies
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/HD TV Shows] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/HD TV Shows
    [/Volumes/DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/Videos] => storage_name://DuneHDD_43c2de36_5497_4ba0_8204_5ab80f57f965/Videos

In your log all movies that you played is in /Volumes/DuneHD...65/ -> /tmp/mnt/...
try to change this area too or use any of the others.

**Comment by wgoldman** [comment]
Thanks for taking a closer look at this, newloran2.  The 6 areas you see listed (3D Movies, DVD Movies, etc.) are the only movie folders I have on the Dune HDD.  I've just re-entered the "storage_name:// . . . " paths again, just as you instructed, for all 6 of these folders in the emplexer preferences.  I've also removed any duplicate movie files from all other networked storage, including the networked computer that is running the Plex Media Server.  I still get the same failure I described above for full-disc rips (.iso or .dvdmedia).  No problem playing .mkv files that are in these same folders.  I've updated my emplexer logs at the same Dropbox folder listed above, if you'd like to take another look.
